### PR TITLE
Adding epochs as a parameter to the constructor for MatrixFactorization class

### PR DESCRIPTION
**Pull request by ssandanshi**
This will make the epochs tunable.

### CODE REVIEW

File: fancyimpute/matrix_factorization.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def __init__(
             self,
             rank=10,
             learning_rate=0.001,
+            epochs = 10000
```
Comment (sergeyf) [inline fancyimpute/matrix_factorization.py]: Please get rid of the spaces around = to conform to the rest of the code.

**Comment by sergeyf** [comment]
Thanks for the PR! Just one little style comment.

**Comment by sergeyf** [comment]
You also forgot a comma, so the builds are failing.

File: fancyimpute/matrix_factorization.py (hunk 1 above)
Comment (ssandanshi) [reply fancyimpute/matrix_factorization.py]: fixed this

**Comment by sergeyf** [comment]
Thanks again!

**Pull request merged**